app/models/concerns/geoblacklight/suppressed_records_search_behavior.rb

```diff
@@ -12,8 +12,13 @@ module SuppressedRecordsSearchBehavior
     # @param [Blacklight::Solr::Request]
     # @return [Blacklight::Solr::Request]
     def hide_suppressed_records(solr_params)
-      # Show child records if searching for a specific source parent
-      return unless blacklight_params.fetch(:f, {})[Settings.FIELDS.SOURCE.to_sym].nil?
+      # Show suppressed records when searching relationships
+      return if blacklight_params.fetch(:f,
+                                        {}).keys.any? do |field|
```

Comment:
issue: this would be easier to read if line 17 was moved up to line 16.

Reply:
Ahhh! Yeah, I agree it looks odd, but this is how Rubocop autocorrected the indentation. 🤷

Reply:
Rubocop. smh.